android-adobe-tracker/src/main/java/io/techery/analytics/tracker/AdobeTracker.java

```diff
@@ -1,19 +1,24 @@
 package io.techery.analytics.tracker;
 
 import android.app.Activity;
+import android.app.Application;
 import android.os.Bundle;
 import com.adobe.mobile.Analytics;
 import com.adobe.mobile.Config;
 import io.techery.janet.analytics.Tracker;
 
 import java.util.Map;
 
-public class AdobeTracker implements Tracker, ActivityLifecycleConsumer {
+public class AdobeTracker implements Tracker, Application.ActivityLifecycleCallbacks {
 
    public static final String KEY = "JANET_ADOBE_TRACKER";
 
    protected boolean appContextSet = false;
 
+   public AdobeTracker(Application application) {
+      application.registerActivityLifecycleCallbacks(this);
```

Comment:
I would rather do this via anonymous class implementation to not expose knowledge about `Application.ActivityLifecycleCallbacks` to publicity. But this is not something very important